src/cmd/services/m3dbnode/config/bootstrap.go

```diff
@@ -70,18 +71,51 @@ type BootstrapConfiguration struct {
 type BootstrapFilesystemConfiguration struct {
 	// NumProcessorsPerCPU is the number of processors per CPU.
 	NumProcessorsPerCPU float64 `yaml:"numProcessorsPerCPU" validate:"min=0.0"`
+
+	// Migration configuration specifies what version, if any, existing data filesets should be migrated to
```

Comment:
Will need  `.` at end of these comments (I believe these get genned to godocs so it's a bit janky if there's no full stop at the end, so not really a concern for comments for the sake of comments, e.g. within a function)

also meganit: `specifies  which version...`

Reply:
Ahh, gotcha. Will make a concerted effort to remember my periods going forward. May be some slip ups though :)